project/src/components/comment-form/comment-form.tsx

```diff
@@ -1,10 +1,26 @@
-import React from 'react';
+import React, { FormEvent } from 'react';
+import { useAppDispatch } from '../../hooks';
+import { addReviewAction } from '../../store/api-actions';
 
-function CommentForm () : JSX.Element {
+const checkCommentValidityLength = (text: string) => {
+  if (text.length < 50 || text.length > 300) {
+    return false;
+  } else
+  {return true;}
```

Comment:
```suggestion
  }

  return true;
```
Что-то форматирование кривое какое то, в принципе `else` можно вообще опустить.